tests/phpunit/CRM/Contribute/BAO/ContributionTest.php

```diff
@@ -820,8 +820,6 @@ public function testcheckLineItems() {
         $e->getMessage()
       );
     }
-
-    $this->assertEquals(3, $params['line_items'][0]['line_item'][0]['financial_type_id']);
```

Comment:
No longer done in the same low-level function (which I intend to remove anyway soon)